ember.md

```diff
@@ -67,6 +67,27 @@ foo: undefined
 foo: null
 ```
 
+- 1.4 **Multi prop components should always have a new line per property**
+
+  + All components written in handlebars that have at least one property should be written in a multi-line format.
```

Comment:
Can you add a "Why" statement to this?
> Why? This structure is to blah blah blah...

Reply:
There's already a why statement for General Structure so I don't think is necessary to add another one.

Reply:
The Why statement in the General Structure explains why we structure our JavaScript code in ember, but not why we as a company have decided to structure our multi-line components this way. For Default Unassigned Properties (section 1.3) we added a Why statement to explain why we made that decision for that standard.

Reply:
Anchor tag is still there but commented out?